third_party/move/move-model/src/symbol.rs

```diff
@@ -65,14 +65,17 @@ impl SymbolPool {
     /// pool. The implementation uses internally a RefCell for storing symbols, so the pool
     /// does not need to be mutable.
     pub fn make(&self, s: &str) -> Symbol {
-        let mut pool = self.inner.borrow_mut();
         let key = Rc::new(s.to_string());
-        if let Some(n) = pool.lookup.get(&key) {
-            return Symbol(*n);
+        {
```

Comment:
Why this change?

Reply:
I had previously made some more significant changes to `SymbolPool` to track the shadow syms, and noted that we don't need to `borrow_mut` the `pool` if we're just returning a known symbol.  This seemed likely to be slower than a read-only `borrow` so I made that change.  It's not a definite win so I'll revert it from this PR since it's unrelated now.